python.d/elasticsearch.chart.py

```diff
@@ -166,13 +189,69 @@
             ['indexing_latency', 'indexing', 'absolute', 1, 1000],
             ['flushing_latency', 'flushing', 'absolute', 1, 1000]
         ]},
+    'index_translog_operations': {
+        'options': [None, 'Translog Operations', 'count', 'translog',
+                    'elastic.index_translog_operations', 'area'],
+        'lines': [
+            ['indices_translog_operations', 'total', 'absolute'],
+            ['indices_translog_uncommitted_operations', 'uncommited', 'absolute']
+        ]},
+    'index_translog_size': {
+        'options': [None, 'Translog Size', 'MB', 'translog',
+                    'elastic.index_translog_size', 'area'],
+        'lines': [
+            ['indices_translog_size_in_bytes', 'total', 'absolute', 1, 1048567],
+            ['indices_translog_uncommitted_size_in_bytes', 'uncommited', 'absolute', 1, 1048567]
+        ]},
+    'index_segments_count': {
+        'options': [None, 'Total Number Of Indices Segments', 'count', 'indices segments',
+                    'elastic.index_segments_count', 'line'],
+        'lines': [
+            ['indices_segments_count', 'segments', 'absolute']
+        ]},
+    'index_segments_memory': {
+        'options': [None, 'Indices Segments Memory Usage', 'MB', 'indices segments',
+                    'elastic.index_segments_memory', 'area'],
+        'lines': [
+            ['indices_segments_memory_in_bytes', 'total', 'absolute', 1, 1048567],
+            ['indices_segments_terms_memory_in_bytes', 'terms', 'absolute', 1, 1048567],
+            ['indices_segments_stored_fields_memory_in_bytes', 'stored_fields', 'absolute', 1, 1048567],
+            ['indices_segments_term_vectors_memory_in_bytes', 'term_vectors', 'absolute', 1, 1048567],
+            ['indices_segments_norms_memory_in_bytes', 'norms', 'absolute', 1, 1048567],
+            ['indices_segments_points_memory_in_bytes', 'points', 'absolute', 1, 1048567],
+            ['indices_segments_doc_values_memory_in_bytes', 'doc_values', 'absolute', 1, 1048567],
+            ['indices_segments_index_writer_memory_in_bytes', 'index_writer', 'absolute', 1, 1048567],
+            ['indices_segments_version_map_memory_in_bytes', 'version_map', 'absolute', 1, 1048567],
+            ['indices_segments_fixed_bit_set_memory_in_bytes', 'fixed_bit_set', 'absolute', 1, 1048567]
```

Comment:
Please replace underscores with spaces in dimension names
like `fixed_bit_set` => `fixed bit set`

Reply:
Done.